### PR TITLE
Feat/list page

### PR DESCRIPTION
**Pull request by MatiasOS**

### CODE REVIEW

File: src/List/index.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import OrigIdInput from '../components/OrigIdInput';
+import Filters from './Filters';
+import { fields } from './mockedData';
+import 'react-bootstrap-table/dist/react-bootstrap-table-all.min.css';
+import Container from '@windingtree/wt-ui-react/lib/components/layout/Container';
+// import config from '../config';
```
Comment (PabloAlice) [inline src/List/index.js]: is this needed?

File: src/List/index.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+const parseDate = (date) => {
+  const year = date.getFullYear();
+  const month = date.getMonth() + 1;
+  const day = date.getDate();
+  return `${year}-${month}-${day}`
+
+}
```
Comment (PabloAlice) [inline src/List/index.js]: Can we use https://github.com/date-fns/date-fns instead of parsing dates manually?

File: src/List/index.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      organizationsData: []
+    };
+  }
+
+  componentDidMount() {
+    fetch(`http://127.0.0.1:8008/organizations`)
+      .then(response => {
+        return response.json()
+      })
+      .then(data => {
```
Comment (PabloAlice) [inline src/List/index.js]: let's use async/await
FYI you can use
`async componentDidMount() {`

File: src/Routes.js [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
+import { Route, Switch, Redirect } from 'react-router-dom';
+import List from './List';
+import OrgIdDetails from './OrgIdDetails'
 import Page404 from './components/Page404';
 
 const Routes = () => (
   <Switch>
-    <Route path="/" exact component={Home} />
+    <Route path={["/orgid/:id"]} exact component={OrgIdDetails} />
+    <Redirect exact from={["/orgid","/orgid/"]} to='/'/>
```
Comment (PabloAlice) [inline src/Routes.js]: I think the from prop could be simplified like `from={'/orgid'}`

Comment (PabloAlice) [reply src/Routes.js]: Also, is this Redirect needed?

File: src/List/index.js [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+    e.preventDefault();
+    const { selectedDirectory, startDate, endDate,sortOrder, sortName } = this.state;
+
+    const segment = `segment=${selectedDirectory}`;
+    const dateCreatedFrom = `dateCreatedFrom=${parseDate(startDate)}`
+    const dateCreatedTo = `dateCreatedTo=${parseDate(endDate)}`
+    const sortingOrder = sortOrder === 'desc' ? '-' : '';
+    const sortingField = `sortingField=${sortingOrder}${sortName}`;
+
+    fetch(`http://127.0.0.1:8008/organizations?${segment}&${dateCreatedFrom}&${dateCreatedTo}&${sortingField}`)
```
Comment (PabloAlice) [inline src/List/index.js]: we probably should consider using axios, maybe in the future.

File: src/List/index.js (hunk 2 above)
Comment (MatiasOS) [reply src/List/index.js]: Done

File: src/List/index.js (hunk 1 above)
Comment (MatiasOS) [reply src/List/index.js]: Yes, to change enviroments

File: src/List/index.js (hunk 3 above)
Comment (MatiasOS) [reply src/List/index.js]: done

File: src/Routes.js (hunk 4 above)
Comment (MatiasOS) [reply src/Routes.js]: Fixed

File: src/List/index.js (hunk 5 above)
Comment (MatiasOS) [reply src/List/index.js]: Yes

**Pull request merged**